cou/steps/backup.py

```diff
@@ -44,9 +44,15 @@ async def backup(model: Model) -> Path:
     await model.run_on_unit(unit_name, f"chmod o+rx {basedir}")
 
     local_file = COU_DATA / os.path.basename(remote_file)
-    logger.info("SCP from  mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
+    logger.info("SCP from mysql-innodb-cluster:%s to %s ...", remote_file, local_file)
     await model.scp_from_unit(unit_name, remote_file, str(local_file))
 
+    # info() stop the current spinner and print the text with
```

Comment:
Thanks for the comment, but I was thinking more of a "why" comment like this:

```suggestion
    # NOTE: a spinner is already running when this function is called,
    # so to cleanly print text, we can use .info(), then restart the spinner.
    # Not using `print` directly,
    # because that will cause corrupted output while the spinner is running.
    # info() stop the current spinner and print the text with
```

Reply:
I am ok with the change but do we need to explain why not using print?

Reply:
To me it's confusing using a global spinner object here - the code makes some assumptions (the global spinner will be running) and reasons for not using alternatives (text will be corrupted by the spinner), so we should clearly write out those assumptions and reasons. :)

Reply:
Updated